### DISCUSSION TITLE
Add integration for dev.to

### DISCUSSION THREAD

**Discussion by davidmyersdev** [davidmyersdev/octo#154] [Ideas]
We write in markdown already. Why not allow users to create draft articles in [dev.to](https://dev.to) with the click of a button?

### Resources

- https://docs.dev.to/api/#operation/createArticle

**Comment by rlnorthcutt** [comment]
Actually, I'd like to see this for ANY MD blog, including GH based pages, static generated sites, etc. I've looked for tools like this, but there aren't any good ones.

**Reply by davidmyersdev** [reply]
I'd still love to see this, but it fell in priority after I pivoted Octo away from publishing and into the second brain (knowledge base) space. I started working on a standalone package for this about 1-2 years ago called [xp](https://github.com/voracious/xp) (short for cross-post) that focused on dev.to, medium.com, and hashnode.com, but the idea was to make it extensible enough for any use case.

**Reply by rlnorthcutt** [reply]
I agree that the "2nd brain" use case is primary, and needs to be the focus. At some point, being able to tie into other systems to share/push MD files from the DB will be a great feature!
